octodns_powerdns/__init__.py

```diff
@@ -72,6 +72,7 @@ def __init__(
         timeout=TIMEOUT,
         soa_edit_api='default',
         mode_of_operation='master',
+        ssl_verify=True,
```

Comment:
Moving ssl_verify up to live by https since they're closely related.

```suggestion
        ssl_verify=True,
        timeout=TIMEOUT,
        soa_edit_api='default',
        mode_of_operation='master',
```